doc/guides/contributing/pull-requests.md

```diff
@@ -134,12 +136,33 @@ to review changes that are split across multiple commits.
 
 ```text
 $ git add my/changed/files
-$ git commit
+$ git commit -s
 ```
 
 Multiple commits often get squashed when they are landed. See the
 notes about [commit squashing](#commit-squashing).
 
+#### Signing off on commits
+
+All commits landed must contain one `Signed-off-by` metadata statement for
+each attributed contributor to the change. The value of this statement must
+match the name and email address configured in [Step 1][].
+
+```text
+Signed-off-by: J. Random User <j.random.user@example.com>
+```
+
+The `Signed-off-by` statement can be added automatically by git by using the
+`-s` (or `--signof`) option when commiting:
```

Comment:
```suggestion
To add the `Signed-off-by` statement with `git`, use the `-s` (or `--signoff`)
option:
```

Note that even if you disregard this suggestion, please at least fix the typo (`--signoff` instead of `--signof`).